src/main/java/com/iridium/iridiumskyblock/database/IslandBan.java

```diff
@@ -17,13 +17,13 @@
 @DatabaseTable(tableName = "island_bans")
 public class IslandBan extends IslandData {
 
-    @DatabaseField(columnName = "id", generatedId = true, canBeNull = false)
+    @DatabaseField(columnName = "id", generatedId = true, canBeNull = false, unique = true, uniqueCombo = true)
```

Comment:
i've tried adding columns but its throws errors, so Orm doesn't recreate|update tables if is exist, ik this not a new column but i guess orm wont update the table 
But u're god -_-